packages/nextjs-template/components/DendronSearch.tsx

```diff
@@ -227,7 +228,21 @@ function DendronSearchComponent(
         : lookupResults.map((noteIndex: NoteIndexProps) => {
             return (
               <AutoComplete.Option key={noteIndex.id} value={noteIndex.fname}>
-                <div>{noteIndex.fname}</div>
+                <Link href={`/notes/${noteIndex.id}`}>
+                  {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
```

Comment:
This is required because of the way nextjs `Link` work. In the resulting DOM there will be only one anchor element so there is no need to fix this warnings.
<img width="667" alt="Screenshot 2022-01-09 at 21 29 50" src="https://user-images.githubusercontent.com/11915087/148695513-83395c1e-b9b0-4462-b138-904581d2f541.png">